app/frontend/src/components/TextField.tsx

```diff
@@ -23,14 +23,20 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+interface TextfieldProps extends OutlinedTextFieldProps {
```

Comment:
This can just use TextFieldProps from material-ui since we have parametrized variant now